src/Resource.js

```diff
@@ -43,6 +43,11 @@ export default class Resource extends Component {
       return;
     }
     document.body.addEventListener("keydown", this.handleKeyDown);
+
+    const script = document.createElement("script");
+    script.src = "https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js";
+    script.async = true;
+    document.body.appendChild(script);
```

Comment:
Oh nice! I assume this script renders the LaTeX?

Reply:
I assume that you rescue the LaTeX in course generation somewhere else.

Reply:
Is this added to every single page?

Reply:
Yep. I'm working on a way to add it only once and call `MathJax.startup.defaultReady()` when something changes.

Reply:
You do not need (or want) to call `MathJax.startup.defaultReady()` unless your configuration has a `startup.ready()` override function, and then only call it from within that function (once).  Is there a MathJax configuration somewhere, or are you using the default settings?

If you want to have MathJax re-typeset the page when something changes, use `MathJax.typesetPromise()` to do so.

Reply:
Got it, thanks!